connect/client_test.go

```diff
@@ -562,7 +607,63 @@ func deleteItem(req *http.Request) (*http.Response, error) {
 	return &http.Response{
 		Status:     http.StatusText(http.StatusNoContent),
 		StatusCode: http.StatusNoContent,
+		Header:     req.Header,
 		Body:       ioutil.NopCloser(&bytes.Buffer{}),
+	}, nil
+}
+
+func generateFile() *onepassword.File {
+	return &onepassword.File{
+		ID:          uuid.New().String(),
+		Name:        "testfile.txt",
+		ContentPath: "/v1/files/xbqdtnehinocwuz23c7l7jiagy/content",
+	}
+}
+
+func getFile(req *http.Request) (*http.Response, error) {
+	if requestFail {
+		json, _ := json.Marshal("Not found")
+		return &http.Response{
+			Status:     http.StatusText(http.StatusNotFound),
+			StatusCode: http.StatusNotFound,
+			Body:       ioutil.NopCloser(bytes.NewReader(json)),
+			Header:     req.Header,
+		}, nil
+	}
+
```

Comment:
This can be removed, since it is never reached in testing. The failure is achieved by calling `respondErr` function in the test case (which is already done)
```suggestion
```